server/controller/users.js

```diff
@@ -24,15 +24,16 @@ class UserController {
    */
   static async registerUser(req, res, next) {
     const {
-      firstName, lastName, email, userName, password, gender
+      firstName, lastName, email, role, userName, password, gender
     } = req.body;
-
+    const userRole = role ? role.toLowerCase() : 'trainee';
     const [user, created] = await User.findOrCreate({
       where: { email },
       defaults: {
         firstName,
         lastName,
         email,
+        role: userRole,
```

Comment:
We don't want to give the user the power to set their own role, so get rid of this line like we discussed